sklearn/model_selection/_validation.py

```diff
@@ -1349,7 +1371,7 @@ def learning_curve(estimator, X, y, *, groups=None,
     parallel = Parallel(n_jobs=n_jobs, pre_dispatch=pre_dispatch,
                         verbose=verbose)
 
-    if shuffle:
+    if shuffle and not stratify:  # if stratify, we shuffle at each iteration
```

Comment:
```suggestion
    if shuffle and not stratify:
        # if stratify, we shuffle at each iteration
```

Reply:
the comment should also indicate where the shuffling will happen

Reply:
@glemaitre, sorry I don't understand what needs to be added here..